discoverygarden/islandora_pdfjs
# Drupal 8 Port

# Basic D8 port

# How should this be tested?

Verify all D7 functionality is accounted for in D8.

Requires https://github.com/discoverygarden/islandora_pdf/pull/4
